packages/storage/amplify_storage_s3/README.md

```diff
@@ -22,3 +22,89 @@ For production use cases please use the latest, non-tagged versions of amplify-f
 ## Getting Started
 
 ### Visit our [Web Site](https://docs.amplify.aws/) to learn more about AWS Amplify.
+
+## Upgrade from Stable Version
+
+All Storage S3 plugin APIs now return an operation object rather than the result object. The operation object contains a `Future` of the result. And the result object contains more information.
```

Comment:
```suggestion
All Storage S3 plugin APIs now return an operation object rather than the result object. The operation object provides more control over the in-flight request, such as cancellation, pause, and resume capabilities (varies by API). The result `Future` can be retrieved via the `.result` property.
```